engine/launcher/src/main/scala/org/enso/launcher/components/ComponentsManager.scala

```diff
@@ -145,7 +146,10 @@ class ComponentsManager(
       }
       .recoverWith {
         case _: ComponentMissingError        => Success(None)
-        case e: LauncherUpgradeRequiredError => Failure(e)
+        case e: LauncherUpgradeRequiredError =>
+          // TODO [RW] consider offering to upgrade automatically (but make sure
```

Comment:
When is this todo going to be resolved?

Reply:
Oh. Thanks for catching it! It has been resolved but it is handled elsewhere and I forgot to remove the comment here.